Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -178,7 +178,15 @@ extension NavigationMapView {
      - parameter coordinate: Current position of the user location.
      */
     public func travelAlongRouteLine(to coordinate: CLLocationCoordinate2D?) {
-        updateFractionTraveled(coordinate: coordinate)
+        guard preLocation != coordinate,
+              let preCoordinate = preLocation,
+              let currentCoordinate = coordinate else { return }
+        
+        let distance = preCoordinate.distance(to: currentCoordinate)
+        let meterPerPixel = getMetersPerPixelAtLatitude(currentCoordinate.latitude, Double(mapView.cameraState.zoom))
+        guard distance >= meterPerPixel else { return }
```

Comment:
Is it possible that due to this `guard` `fractionTraveled` will not be updated on low zoom levels and it can malfunction completing the route (i.e. `fractionTraveled >= 1` check)? What if user has to travel less distance to his destination than fits into 1 pixel?